apps/src/gamelab/GameLabSprite.js

```diff
@@ -251,10 +248,12 @@ var AABBops = function (p5Inst, type, target, callback) {
               callback.call(this, this, other);
           }
         }
-      else if(type === 'collide' || type === 'bounce')
+      else if(type === 'collide' || type === 'bounce' || type == 'bounceOff')
```

Comment:
It strikes me that this change will make it that much harder to update our local copy of p5.play any time we want to pull a new version.  Not sure there's anything we can do about that, just wondering if there's a way to mitigate it.